### ISSUE TITLE
htop does not run

### ISSUE THREAD

**Issue by weiszr** [tbodt/ish#273]
I successfully install htop, but nothing is shown (no screen output at all).

**Comment by Mrc527** [comment]
I can confirm, same behavior on iOS 13 as well.

**Comment by weiszr** [comment]
Yup...still does not work.



______________________________________________________________
Robert Weiss
*Professor of Natural Hazards*
Department of Geosciences
Virginia Tech
4044 Derring Hall (0420)
Blacksburg, VA 24061, U.S.A.

Office: 1068A Derring
Phone:+1-540-231-2334
Fax: +1-540-231-3386


Research Web: http://coastalhazardsvt.weebly.com
Education Web: https://fralin.vt.edu/DRRM.html


Twitter: @VTCoastal

--


--

**Comment by LinuxOnboard** [comment]
I believe it is a problem with ncurses.

**Comment by LinuxOnboard** [comment]
What are some ncurses commands/programs that you tested?

**Comment by jaderfeijo** [comment]
I don’t believe this is related to ncurses, I’ve been able to install and use tmux which so far has worked for me.

**Comment by Assfugil** [comment]
it won't respond to SIGKILL either.

**Comment by StewAlexander-com** [comment]
Anyone really good with stack traces? Maybe this might shed light on what's going on with htop?

**Comment by 62f** [comment]
@tbodt, hello. I made these changes:

- Loading per cpu usage
- List of alive pids
- More `TASK_MAY_BLOCK`

__Originally posted by @nimelehin in https://github.com/ish-app/ish/pull/1066#issuecomment-742613261__

https://discord.com/channels/508839261924229141/510973900209913857/811252086557114378 htop also gives me ```No btime in /proc/stat: No such file or directory``` for Alpine 3.13.1

yes htop and non-busybox tops are broken

**Comment by 62f** [comment]
![BFE151AC-70F3-4E3A-9E82-B698289F679F](https://user-images.githubusercontent.com/68332197/117942991-aae95d80-b2d9-11eb-96d0-f9dc408c7384.jpeg)

**Comment by duckfromdiscord** [comment]
im having this issue too, build 1.2.3 298
